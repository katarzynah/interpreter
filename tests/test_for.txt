program f;
var i, k: int;

begin
    k := 3;
    for i := 1 to 10 do
    begin
        print(k * i + 3);
    end;
end.