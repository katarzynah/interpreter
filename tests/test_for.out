6
9
12
15
18
21
24
27
30
33
[(fromList [(Ident "i",11),(Ident "k",3)],fromList [])]
